### PR TITLE
twitch-tui: init at 1.6.0

### PR DESCRIPTION
**Pull request by tgharib**
###### Description of changes

https://github.com/Xithrius/twitch-tui

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tgharib** [comment]
Result of `nixpkgs-review pr 182403` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>twitch-tui</li>
  </ul>
</details>

File: pkgs/applications/networking/instant-messengers/twitch-tui/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchFromGitHub {
+    owner = "Xithrius";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-144yn/QQPIZJOgqKFUWjB7KCmEKfNpj6XjMGhTpQdEQ=";
+  };
+
+  OPENSSL_LIB_DIR = "${lib.getLib openssl}/lib";
+  OPENSSL_INCLUDE_DIR = "${openssl.dev}/include";
```
Comment (fpletz) [inline pkgs/applications/networking/instant-messengers/twitch-tui/default.nix:15]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ openssl ];
```

That's what pkg-config is for. :smiley_cat:

File: pkgs/applications/networking/instant-messengers/twitch-tui/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ openssl, lib, fetchFromGitHub, rustPlatform }:
```
Comment (fpletz) [inline pkgs/applications/networking/instant-messengers/twitch-tui/default.nix:1]:
```suggestion
{ lib, fetchFromGitHub, rustPlatform, pkg-config, openssl }:
```

**Review by fpletz — CHANGES_REQUESTED** [review]

**Comment by tgharib** [comment]
@fpletz I went ahead and merged your changes.

**Comment by tgharib** [comment]
Result of `nixpkgs-review pr 182403` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>twitch-tui</li>
  </ul>
</details>

**Review by fpletz — APPROVED** [review]
One more thing I forgot: The version should be without the v. That also changed the cargoHash. Pushed into your branch.

**Comment by tgharib** [comment]
Cool. Nothing needed from my end right?

**Comment by fpletz** [comment]
Yup, was just waiting for ofborg to finish. Thanks! :smile:

**Pull request merged**